src/ToDo.UI/App.xaml.host.cs

```diff
@@ -112,7 +112,7 @@ string ResourceKey(string keyPath)
 			/// Dialogs and Flyouts
 			new ViewMap<AddTaskFlyout, AddTaskViewModel>(),
 			new ViewMap<AddListFlyout, AddListViewModel>(),
-			new ViewMap<ExpirationDateFlyout, ExpirationDateViewModel>(),
+			new ViewMap<ExpirationDateFlyout, ExpirationDateViewModel.BindableExpirationDateViewModel>(),
```

Comment:
This isn't correct - please follow the pattern of the other viewmaps. The BindableExpirationDateViewModel needs to be mapped in the ReactiveClasses.cs.